blockchain/objects/state.go

```diff
@@ -36,17 +36,18 @@ type MonitorState struct {
 }
 
 // EthDKGPhase is used to indicate what phase we are currently in
-type EthDKGPhase int
+type EthDKGPhase uint8
 
 // These are the valid phases of ETHDKG
 const (
-	Registration EthDKGPhase = iota
+	RegistrationOpen EthDKGPhase = iota
```

Comment:
iota + 1

Reply:
force default value is invalid

Reply:
! investigate, if necessary preserve.

Reply:
can we get through the bindings?

Reply:
bindings don't give us this information yet https://github.com/ethereum/EIPs/issues/47